Userland/Applications/HexEditor/HexEditorWidget.cpp

```diff
@@ -44,14 +44,28 @@ HexEditorWidget::HexEditorWidget()
     m_search_results = *find_descendant_of_type_named<GUI::TableView>("search_results");
     m_search_results_container = *find_descendant_of_type_named<GUI::Widget>("search_results_container");
 
-    m_editor->on_status_change = [this](int position, HexEditor::EditMode edit_mode, int selection_start, int selection_end) {
+    auto update_statusbar_selection_and_offset_info = [this](int selection_start, int selection_end) {
+        if (m_show_selection_information_as_hex)
+        {
```

Comment:
Even though this is remove with the next commit, please follow the code style and put if-braces on the same line.